diff --git a/dist/index.js b/dist/index.js
index 996b0d16dac39667cc25496e52adcc9dd2b2befa..a4c9f5ba3a20967d9a561fcc73178d9e84f48279 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -245,7 +245,7 @@ var codePlugin = (md, {
     const info = token.info ? md.utils.unescapeAll(token.info).trim() : "";
     const language = resolveLanguage(info);
     const languageClass = `${options.langPrefix}${language.name}`;
-    const code = options.highlight?.(token.content, language.name, "") || md.utils.escapeHtml(token.content);
+    const code = options.highlight?.(token.content, language.name, info || "") || md.utils.escapeHtml(token.content);
     token.attrJoin("class", languageClass);
     let result = code.startsWith("<pre") ? code : `<pre${slf.renderAttrs(token)}><code>${code}</code></pre>`;
     const useVPre = resolveVPre(info) ?? vPreBlock;
