diff --git a/dist/index.js b/dist/index.js
index de136bbd25d01677b7b86b6654b780765b9b7cec..b0ff18b8888fe4e691e8c8c2ee3db6ba9fd35ed2 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -267,7 +267,7 @@ var codePlugin = (md, {
     const info = token.info ? md.utils.unescapeAll(token.info).trim() : "";
     const language = resolveLanguage(info);
     const languageClass = `${options.langPrefix}${language.name}`;
-    const code = options.highlight?.(token.content, language.name, "") || md.utils.escapeHtml(token.content);
+    const code = options.highlight?.(token.content, language.name, info || "") || md.utils.escapeHtml(token.content);
     token.attrJoin("class", languageClass);
     let result = code.startsWith("<pre") ? code : `<pre${slf.renderAttrs(token)}><code>${code}</code></pre>`;
     const useVPre = resolveVPre(info) ?? vPreBlock;
