diff --git a/dist/floating-vue.mjs b/dist/floating-vue.mjs
index d54beb90f1fd341fb0aa41e1503afdbbece12cbb..c6b8ed8d5f0213700887509ede99b37118c9b4c0 100644
--- a/dist/floating-vue.mjs
+++ b/dist/floating-vue.mjs
@@ -947,6 +947,7 @@ function mt(e, t, o, i, s, r) {
   return f(), T("div", {
     id: e.popperId,
     ref: "popover",
+    'data-allow-mismatch': '',
     class: J(["v-popper__popper", [
       e.themeClass,
       e.classes.popperClass,
@@ -966,7 +967,7 @@ function mt(e, t, o, i, s, r) {
       position: e.result.strategy,
       transform: `translate3d(${Math.round(e.result.x)}px,${Math.round(e.result.y)}px,0)`
     } : void 0),
-    "aria-hidden": e.shown ? "false" : "true",
+    "aria-hidden": e.shown || e.autoHide ? "false" : "true",
     tabindex: e.autoHide ? 0 : void 0,
     "data-popper-placement": e.result ? e.result.placement : void 0,
     onKeyup: t[2] || (t[2] = je((a) => e.autoHide && e.$emit("hide"), ["esc"]))
@@ -1249,7 +1250,7 @@ function wt(e, t, o, i, s, r) {
       onResize: j,
       classes: V,
       result: Ee
-    }) => [
+    } = {}) => [
       A(e.$slots, "default", {
         shown: u,
         show: F,
@@ -1258,6 +1259,7 @@ function wt(e, t, o, i, s, r) {
       ve(p, {
         ref: "popperContent",
         "popper-id": l,
+        'data-allow-mismatch': '',
         theme: e.finalTheme,
         shown: u,
         mounted: L,
@@ -1401,6 +1403,7 @@ function _t(e, t, o, i, s, r) {
           "v-popper--tooltip-loading": e.loading
         }),
         "popper-id": l,
+        'data-allow-mismatch': '',
         theme: e.theme,
         shown: u,
         mounted: L,
